projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -861,13 +878,27 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
     public additionalTemplateContext: any;
 
     /**
-     * @hidden
+     * Emitted when the column width changes.
+     *
+     * ```html
+     * <igx-column (widthChange)="widthChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
      */
     @Output()
     public widthChange = new EventEmitter<string>();
```

Comment:
I dont see a case that this is useful since this is pure column configurable and there is no way that a user can trigger this with an interaction

Reply:
What about when resizing a column?